diff -uNr luasql-2.3.0_org/config luasql-2.3.0/config
--- luasql-2.3.0_org/config	2013-11-29 23:24:58.096295612 +0800
+++ luasql-2.3.0/config	2013-11-29 23:28:03.968291549 +0800
@@ -1,30 +1,30 @@
 # $Id: config,v 1.10 2008/05/30 17:21:18 tomas Exp $
 
 # Driver (leave uncommented ONLY the line with the name of the driver)
-T= mysql
+#T= mysql
 #T= oci8
 #T= odbc
 #T= postgres
 #T= sqlite
-#T=sqlite3
+T=sqlite3
 #T=firebird
 
 # Installation directories
 
 # Default prefix
-PREFIX = /usr/local
+PREFIX = /usr
 
 # System's libraries directory (where binary libraries are installed)
-LUA_LIBDIR= $(PREFIX)/lib/lua/5.1
+LUA_LIBDIR= $(PREFIX)/lib/lua/5.2
 
 # System's lua directory (where Lua libraries are installed)
-LUA_DIR= $(PREFIX)/share/lua/5.1
+LUA_DIR= $(PREFIX)/share/lua/5.2
 
 # Lua includes directory
 LUA_INC= $(PREFIX)/include
 
 # Lua version number (first and second digits of target version)
-LUA_VERSION_NUM= 501
+LUA_VERSION_NUM= 502
 
 # OS dependent
 LIB_OPTION= -shared #for Linux
@@ -37,8 +37,8 @@
 ######## MySQL
 #DRIVER_LIBS= -L/usr/local/mysql/lib -lmysqlclient -lz
 #DRIVER_INCS= -I/usr/local/mysql/include
-DRIVER_LIBS= -L/usr/lib -lmysqlclient -lz
-DRIVER_INCS= -I/usr/include/mysql
+#DRIVER_LIBS= -L/usr/lib -lmysqlclient -lz
+#DRIVER_INCS= -I/usr/include/mysql
 ######## Oracle OCI8
 #DRIVER_LIBS= -L/home/oracle/OraHome1/lib -lz -lclntsh
 #DRIVER_INCS= -I/home/oracle/OraHome1/rdbms/demo -I/home/oracle/OraHome1/rdbms/public
@@ -51,8 +51,8 @@
 #DRIVER_LIBS= -lsqlite
 #DRIVER_INCS=
 ######## SQLite3 
-#DRIVER_LIBS= -L/opt/local/lib -lsqlite3
-#DRIVER_INCS= -I/opt/local/include
+DRIVER_LIBS= -L$(PREFIX)/lib -lsqlite3
+DRIVER_INCS= -I$(PREFIX)/include
 ######## ODBC
 #DRIVER_LIBS= -L/usr/local/lib -lodbc
 #DRIVER_INCS= -DUNIXODBC -I/usr/local/include
@@ -62,7 +62,6 @@
 
 WARN= -Wall -Wmissing-prototypes -Wmissing-declarations -ansi -pedantic
 INCS= -I$(LUA_INC)
-CFLAGS= -O2 $(WARN) -fPIC $(DRIVER_INCS) $(INCS) -DLUASQL_VERSION_NUMBER='"$V"' $(DEFS)
-CC= gcc
+CFLAGS= -O2 $(WARN) -fPIC $(DRIVER_INCS) -DLUASQL_VERSION_NUMBER='"$V"' $(DEFS)

# $Id: config,v 1.10 2008/05/30 17:21:18 tomas Exp $
