diff -uNr rings-v_1_3_0_org/config rings-v_1_3_0/config
--- rings-v_1_3_0_org/config	2013-11-30 00:38:53.992198653 +0800
+++ rings-v_1_3_0/config	2013-11-30 00:39:22.304198035 +0800
@@ -1,15 +1,15 @@
 # Installation directories
 
 # Default prefix
-PREFIX ?= /usr/local
+PREFIX ?= /usr
 
 DESTDIR ?= /
 
 # System's libraries directory (where binary libraries are installed)
-LUA_LIBDIR ?= $(PREFIX)/lib/lua/5.1
+LUA_LIBDIR ?= $(PREFIX)/lib/lua/5.2
 
 # System's lua directory (where Lua libraries are installed)
-LUA_DIR ?= $(PREFIX)/share/lua/5.1
+LUA_DIR ?= $(PREFIX)/share/lua/5.2
 
 # Lua includes directory
 LUA_INC ?= $(PREFIX)/include
@@ -23,7 +23,6 @@
 # Compilation directives
 WARN ?= -O2 -Wall -fPIC -W -Waggregate-return -Wcast-align -Wmissing-prototypes -Wnested-externs -Wshadow -Wwrite-strings -Wpointer-arith -pedantic
 INCS ?= -I$(LUA_INC)
-CFLAGS ?= $(WARN) $(INCS)
-CC ?= gcc
+CFLAGS ?= $(WARN)
 
 # $Id: config,v 1.7 2007/10/29 22:51:39 carregal Exp $
