diff -uNr git_org/CMakeLists.txt git/CMakeLists.txt
--- git_org/CMakeLists.txt	2013-11-30 12:37:37.155256079 +0800
+++ git/CMakeLists.txt	2013-11-30 12:39:40.307253387 +0800
@@ -9,9 +9,9 @@
 
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
 
-set(INSTALL_LMOD ${CMAKE_INSTALL_PREFIX}/share/lua/5.1 CACHE PATH
+set(INSTALL_LMOD ${CMAKE_INSTALL_PREFIX}/share/lua/5.2 CACHE PATH
 				"Directory to install Lua source modules (configure lua via LUA_PATH)")
-set(INSTALL_CMOD ${CMAKE_INSTALL_PREFIX}/lib/lua/5.1 CACHE PATH
+set(INSTALL_CMOD ${CMAKE_INSTALL_PREFIX}/lib/lua/5.2 CACHE PATH
 				"Directory to install Lua binary modules (configure lua via LUA_CPATH)")
 set(ZMQ_PATH "" CACHE PATH
 				"Directory to libzmq. (by default use pkg-config to detect path)")
